src/amchart/Area.js

```diff
@@ -6,35 +11,73 @@
         root.amchart_Area = factory(root.d3, root.amchart_CommonSerial, root.amcharts, root.api_INDChart);
     }
 }(this, function(d3, CommonSerial, AmCharts, INDChart) {
+    /**
+     * @class amchart_Area
+     * @extends amchart_CommonSerial
+     * @extends api_INDChart
```

Comment:
most templates will not compile the interfaces correctly so this was needed to make sure the interfaces methods and the interface it self shows up.